### ISSUE TITLE
UI: Suggestions Text

### ISSUE THREAD

**Issue by sarahghp** [axodotdev/harbor#29]
Include text to preface suggestions that makes the status clear. One way might be to insert between toggle and text either:
_Toggle on to agree to text_ or _Currently this package is marked as_ or something like that.

**Comment by SaraVieira** [comment]
This is also done no?

**Comment by sarahghp** [comment]
No, we haven't done anything about this. The toggle is big. But this wasn't on our list, so I think we can leave it for now.
